<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>103</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="AddButton_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>160</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Показать</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ShowlineEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowButton_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>110</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить последний элемент</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ShowlineEdit_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ShowlineEdit_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowButton_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Заменить </string>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowButton_4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Увеличить емкость на</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ShowlineEdit_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>210</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
